js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@polybase/polylang",
-  "version": "0.4.10",
```

Comment:
We have automate CI for bumping, so no need to bump manually. Can you revert this.

Reply:
Done!